src/modelize/modelize_property.nit

```
@@ -1347,11 +1347,13 @@ redef class AAttrPropdef
 			if nexpr != null then
 				if nexpr isa ANewExpr then
 					mtype = modelbuilder.resolve_mtype_unchecked(mmodule, mclassdef, nexpr.n_type, true)
-				else if nexpr isa AIntExpr then
-					var cla = modelbuilder.try_get_mclass_by_name(nexpr, mmodule, "Int")
-					if cla != null then mtype = cla.mclass_type
-				else if nexpr isa AByteExpr then
-					var cla = modelbuilder.try_get_mclass_by_name(nexpr, mmodule, "Byte")
+				else if nexpr isa AIntegerExpr then
+					var cla: nullable MClass = null
+					if nexpr.value isa Int then
+						cla = modelbuilder.try_get_mclass_by_name(nexpr, mmodule, "Int")
+					else if nexpr.value isa Byte then
+						cla = modelbuilder.try_get_mclass_by_name(nexpr, mmodule, "Byte")
+					end
```

`else abort` to fail early, thus simplify, future evolution